### DISCUSSION TITLE
my application always show up the 'Setup' page after starting up

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#12743] [Q&A]
I have two applications,  one named `Read`, one named `Admin`, which use the same content in `App_Data`.  Weeks before, they shared the same `App_Data` dir ( by setting the env var `ORCHARD_APP_DATA`), then I split the two apps to dif servers, the content in `App_Data`  are copied from one app to another.

Every time I reinstall my applications, I start the `Admin` and fill up the `Setup` form and wait it complete the setup process. Then I copy the newly produced `App_data` to `Read` application, start up the `Read` application, it works well.

Days before, I incidently run `git clean -df -x` and I should reinstall my applications.  The `Admin` went well, but when I start the `Read` application, it always show me the `Setup` page. If I filled the form, it always went wrong.

how I could  debug this?

---

by the way, this time a special thing happened. After setting up the `Admin` application, the normally used themes  `Me.Admin.Theme` and `Me.Read.Theme` were not enabled by default, I had to mannully enable them.   Before, they would be enabled auto.

**Comment by hishamco** [comment]
Don't forgot the `App_Data` contains metadata about the tenants and their status, try to look into `tenants.json`
